workbench/routes/api.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+
+Route::post('agent-auth', static fn () => [
+    'token' => 'NIGHTWATCH_TOKEN',
+    'expires_in' => 60,
+    'refresh_in' => 30,
+    'ingest_url' => 'http://127.0.0.1:8000/api/ingest',
+]);
+
+Route::post('ingest', static function () {
+    if (false) {
+        return response()->json([
+            'message' => 'Exceeded quota',
+        ], 403);
+    }
+
+    if (false) {
+        return response()->json([
+            'message' => 'Invalid body encoding',
+        ], 403);
+    }
+
+    if (false) {
+        return response()->json([
+            'message' => 'Invalid JSON',
+        ], 403);
+    }
+
+    if (false) {
+        return response()->json([
+            'message' => 'Invalid body',
+        ], 403);
+    }
+
+    if (false) {
+        return response()->json([
+            'message' => 'Invalid records',
+        ], 422);
+    }
+
+    if (false) {
+        return response()->json([
+            'message' => 'Unexpected error',
+        ], 500);
+    }
```

Comment:
It's great having the test ingest endpoint! Are we implementing these conditions later?

Reply:
I've been changing the `false` to `true` manually as the requests are being made to see how the agent reacts to different ingest scenarios.

I wasn't planning on making it any more advanced than that for now.